RecoHI/HiMuonAlgos/python/HiRegitMuonSeededStep_cff.py

```diff
@@ -169,18 +169,21 @@
          ) #end of vpset
 )
 
-hiRegitMuonSeededStepCore = cms.Sequence(
-      hiRegitMuonSeededSeedsInOut + hiRegitMuonSeededTrackCandidatesInOut + hiRegitMuonSeededTracksInOut +
-      hiRegitMuonSeededSeedsOutIn + hiRegitMuonSeededTrackCandidatesOutIn + hiRegitMuonSeededTracksOutIn 
+hiRegitMuonSeededStepCoreTask = cms.Task(
+      hiRegitMuonSeededSeedsInOut , hiRegitMuonSeededTrackCandidatesInOut , hiRegitMuonSeededTracksInOut ,
+      hiRegitMuonSeededSeedsOutIn , hiRegitMuonSeededTrackCandidatesOutIn , hiRegitMuonSeededTracksOutIn 
       )
-hiRegitMuonSeededStepExtra = cms.Sequence(
-      hiRegitMuonSeededTracksInOutSelector +
+hiRegitMuonSeededStepCore = cms.Sequence(hiRegitMuonSeededStepCoreTask)
+hiRegitMuonSeededStepExtraTask = cms.Task(
+      hiRegitMuonSeededTracksInOutSelector ,
       hiRegitMuonSeededTracksOutInSelector
       )
+hiRegitMuonSeededStepExtra = cms.Sequence(hiRegitMuonSeededStepExtraTask)
```

Comment:
This was only needed here internally: sequence not needed any more now, please remove